.gitignore

```diff
@@ -8,3 +8,4 @@ build
 *pyc
 *~
 \#*
+.DS_Store
```

Comment:
Consider adding these OS or editor specific files to your user gitignore, e.g.:

```
echo .DS_STORE >> ~/.gitignore
git config --global core.excludesfile ~/.gitignore
```

https://stackoverflow.com/questions/7335420/global-git-ignore
```

Reply:
Thanks, that's useful indeed!